src/main/java/com/jelly/farmhelperv2/feature/impl/AutoSprayonator.java

```diff
@@ -319,16 +332,48 @@ public void onClientTick(TickEvent.ClientTickEvent event) {
                 String sprayType = StringUtils.stripControlCodes(sprayTypeLine.split(": ")[1]);
                 if (sprayType.equals(sprayItem.getItemName())) {
                     sprayState = AUTO_SPRAYONATOR_STATE.CHECK_ITEM;
+                    if (rotationState == 1) {
+                        RotationHandler.getInstance().easeTo(
+                                new RotationConfiguration(
+                                        new Rotation(prevRotation[0], prevRotation[1]),
+                                        FarmHelperConfig.getRandomRotationTime(), () -> rotationState = 0
```

Comment:
If you put callback there, which is different than server side's ease back to client side rotation, you need to also call `reset()` method. Otherwise the field `rotating` won't ever be false

Reply:
skrill issue